cloudant-client/src/main/java/com/cloudant/client/api/query/Operation.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2017 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.cloudant.client.api.query;
+
+import static com.cloudant.client.internal.query.Helpers.quoteCurly;
+
+// operation - used to combine an operator such as "$and" and a series of operations or expressions
+// on the rhs
+public class Operation implements Selector {
+
+    private final String op;
+    private final Selector[] rhs;
+
+    private Operation(String op, Selector... rhs) {
+        this.op = op;
+        this.rhs = rhs;
+    }
+
+    public static Operation and(Selector... rhs) {
```

Comment:
These are more obvious than the `Expression` names, but perhaps we should also have javadoc

Reply:
Done in f134433